src/flick/lst/views.py

```diff
@@ -1,3 +1,160 @@
+from django.contrib.auth.models import User
+from django.conf import settings as dj_settings
 from django.shortcuts import render
 
-# Create your views here.
+from rest_framework import generics, mixins, status, viewsets
+from rest_framework.response import Response
+
+from .models import Lst
+from .serializers import LstSerializer
+from api import settings as api_settings
+from api.utils import failure_response, success_response
+from show.models import Show
+from user.models import Profile
+
+import json
+import re
+
+
+class LstList(generics.ListCreateAPIView):
+
+    queryset = Lst.objects.all()
+    serializer_class = LstSerializer
+
+    permission_classes = api_settings.CONSUMER_PERMISSIONS
+
+    def list(self, request):
+        """See all possible lists."""
+        # TODO: only allow users who are owners / collaborators see lists
+        self.serializer_class = LstSerializer
+        return super(LstList, self).list(request)
+
+    def post(self, request):
+        """Create a list."""
+        data = json.loads(request.body)
+        lst_name = data.get("lst_name")
+        lst_pic = data.get("lst_pic")
+        is_favorite = data.get("is_favorite", False)
+        is_private = data.get("is_private", False)
+        is_watched = data.get("is_watched", False)
+        collaborator_ids = data.get("collaborators", [])
+        show_ids = data.get("shows", [])
+
+        lst = Lst()
+        lst.lst_name = lst_name
+        lst.lst_pic = lst_pic
+        lst.is_favorite = is_favorite
+        lst.is_private = is_private
+        lst.is_watched = is_watched
+        profile = Profile.objects.get(user=request.user)
+        lst.owner = profile
+        lst.save()
+
+        for c_id in collaborator_ids:
+            collaborator = User.objects.get(pk=c_id)
```

Comment:
is user id just its pk? just a question that I am confused about : )

Reply:
yes!!! pk means primary key, and you can also use `id` -- we should probably keep this consistent oops I'll make an issue that'll revert all `pk`s to `id`s